tests/Traces/README

```diff
@@ -1,13 +1,32 @@
-Traces from Zeek:
-    - dns53.pcap
-    - http-post.pcap (post.trace) 
-    - png.pcap (pipelined-requests.trace)
-    - ftp-pe.pcap (pe/pe.trace, pruned a single file transfer)
-    
-Traces from https://wiki.wireshark.org/SampleCaptures:
-    - dhcp.pcap
-    - tftp_rrq.pcap
-    - tftp_wrq.pcap
+The test suite comes with a set of traces collected from a variety of
+places that we document below. While these traces are all coming from
+public sources, please note that they may carry their own licenses.
+We collect them here for convenience only.
 
+DHCP
+- [dhcp.pcap](https://gitlab.com/wireshark/wireshark/-/wikis/uploads/__moin_import__/attachments/SampleCaptures/dhcp.pcap)
 
+DNS
+- [dns53.pcap](https://github.com/zeek/zeek/blob/master/testing/btest/Traces/dns53.pcap)
 
+HTTP
+- [http-post.pcap](https://github.com/zeek/zeek/blob/master/testing/btest/Traces/http/post.trace)
+
+OpenVPN
+- [openvpn-tcp-tls-auth.pcap](https://wiki.wireshark.org/SampleCaptures?action=AttachFile&do=get&target=OpenVPN_TCP_tls-auth.pcapng)
+- [openvpn.pcap](https://wireshark-bugs.wireshark.narkive.com/IwG4sfj5/bug-8240-new-dissector-for-the-openvpn-protocol)
+- [openvpn_tcp_nontlsauth.pcap](https://wireshark-bugs.wireshark.narkive.com/IwG4sfj5/bug-8240-new-dissector-for-the-openvpn-protocol)
```

Comment:
It would be nice to point to the specific attachment while we still have some context as there seem to be multiple versions. Searching for _Created attachment_ in the bug report yields links to the different attachments.

Reply:
Nice, I hadn't see the attachment links. So I found `openvpn_tcp_nontlsauth.pcap` now, but not `openvpn.pcap`; none of the attachment to that ticket seems to match the latter.

Reply:
Hmm, not ideal, but looks like we cannot do much here now.